microshift_install/microshift-install-rhel-for-edge.adoc

```diff
@@ -1,8 +1,23 @@
 :_content-type: ASSEMBLY
 [id="microshift-install-rhel-for-edge"]
-= Installing MicroShift in a RHEL for Edge Image
+= Embedding MicroShift into a {op-system-ostree} image
 include::_attributes/attributes-microshift.adoc[]
 :context: microshift-install-rhel-for-edge
 toc::[]
 
-// The content of this file should be based on https://github.com/openshift/microshift/blob/main/docs/rhel4edge_iso.md
+You can embed MicroShift into a {op-system-first} for Edge 8.7 image.
```

Comment:
Suggest replacing
* {op-system-first} for Edge --> {op-system-ostree-first}
* 8.7 --> {op-system-version}